aws/resource_aws_spot_fleet_request.go

```diff
@@ -256,6 +256,30 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 							Computed: true,
 							ForceNew: true,
 						},
+						"tag": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"key": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+
+									"value": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+								},
+							},
+							Set: spotFleetTagToHash,
+						},
+
+						"tags": {
```

Comment:
The class for aws-sdk is `SpotFleetTagSpecification` with uses `key` and `value`. I just attempted to follow along with the aws model without making too many modifications.

```
tags = [
  {
    key = "key"
    value = "${var.name}"
  }
]
```